### PR TITLE
create CONTRIBUTING.md for master-2.4

### PR DESCRIPTION
**Pull request by warpech**
Similar to PR #15 but for `master-2.4`

@tomalec, @alshakero pls review.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Currently, the developer process looks like follows:
+
+1. Commit changes to individual client-side package's repo, like https://github.com/Starcounter/starcounter-include, and release a version there,
+2. Bump files in a feature branch started from `master` in [this repo](https://github.com/Starcounter/StarcounterClientFiles) either,
+   - by updating `bower.json` and running `bower install`,
+   - or `calling `bower install --save packagename[#version]`
+3. Update `bower-list.txt` by calling `bower list > bower-list.txt`
+4. Make a PR for someone to review the complete set of changes.
+5. As soon as the PR is merged, release new version (2.x) of StarcounterClientFiles (remeber to write the release notes)
```
Comment (alshakero) [inline CONTRIBUTING.md:21]: remember*

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+6. Publish a package of this version to App Warehouse (built with Starcounter 2.4)
+7. Copy manually to a custom branch at level1 (forked from `develop`)
+8. Build level1 branch in TeamCity
+9. Build KitchenSink, Website and CompositionProvider with custom level1 build (pick 2.4 task)
+10. If all tests pass, create a PR to level1 (remember to bump `CHANGELOG-Develop.md`)
+11. You don't need a reviewer for this PR. Simply do something similar to what @warpech did in https://github.com/Starcounter/level1/pull/4420 and give link to the PR in StarcounterClientFiles where it was reviewed in more detail.
```
Comment (alshakero) [inline CONTRIBUTING.md:27]: and give **the** link to the PR in StarcounterClientFiles

**Review by alshakero — COMMENTED** [review]
I'll take care of those comments. Sorry for the delayed review.

**Comment by alshakero** [comment]
Comments applied in https://github.com/Starcounter/StarcounterClientFiles/commit/139311a432a3fe7a79ca1922a57a4bbb70798cbf and https://github.com/Starcounter/StarcounterClientFiles/commit/ba01cff6c521139cde245f7884c61df6a5b5a642.

**Comment by warpech** [comment]
Thank you!

**Pull request merged**